src/components/LoginScreen/index.js

```diff
@@ -1,37 +1,65 @@
-import React from "react";
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import appSchema from '../../validations/app';
+import LanguageContainer from '../../components/LanguageContainer';
+import useForm from 'react-hook-form';
+import Validation from '../../components/Validation';
 import { withRouter } from "react-router";
-import { useTranslation } from "react-i18next";
+
 import styles from "./index.module.scss";
 
 const LoginScreen = props => {
   const { t } = useTranslation();
+  const { register, handleSubmit, errors, triggerValidation, setError } = useForm({
+    validationSchema: appSchema(t),
+    mode: 'onBlur',
+  });
+
+  useEffect(() => {
+    return () => errors && triggerValidation();
+  }, [t]);
 
-  const handleFormSubmit = event => {
-    event.preventDefault();
+  const onSubmit = data => {
+    setError(
```

Comment:
Hmmmm, do we need to manually do this every time?

Reply:
No, I just left an example here in case of if there be some values that are valid but shouldn't be in request, like if we allow phone number input to be without +374 but we want a to show error/warning without doing a request

Reply:
WE need clean examples